### DISCUSSION TITLE
Release Version 0.43.8 - Love Shack 23

### DISCUSSION THREAD

**Discussion by sgoggins** [chaoss/augur#2120] [General]
## What's Changed
* Add bash in the container by @mscherer in https://github.com/chaoss/augur/pull/2112
* updated CONTRIBUTING.md by @WhiteWolf47 in https://github.com/chaoss/augur/pull/2111
* Improved Documentation Format specifically heading and text structure and text  by @glowreeya-01 in https://github.com/chaoss/augur/pull/2118

## New Contributors
* @WhiteWolf47 made their first contribution in https://github.com/chaoss/augur/pull/2111
* @glowreeya-01 made their first contribution in https://github.com/chaoss/augur/pull/2118

**Full Changelog**: https://github.com/chaoss/augur/compare/v0.43.8...v043.8

<hr /><em>This discussion was created from the release <a href='https://github.com/chaoss/augur/releases/tag/v043.8'>Release Version 0.43.8 - Love Shack 23</a>.</em>

**Comment by WhiteWolf47** [comment]
Glad to become a part of this amazing community :)
